docs/provision_hypershift_clusters_by_mce.md

```diff
@@ -101,6 +101,8 @@ stringData:
   aws_access_key_id:      # Value, required
 ```
 
+The `ssh-publickey` and `ssh-privatekey`, if provided, are used to access the worker nodes of the hosted cluster. The SSH Key in the `HostedCluster.Spec` or `HostedDeployment.Spec` takes procedence if they are provided.
+
 You can create this secret using the multicluster console (embedded within the OpenShift console) or via the CLI, both options are shown below:
```

Comment:
This is only true fort he Cloud Provider secret, not individual secrets that you mention in Ln 104, that would be from the OpenShift > Workload > Secret console.

Reply:
That is talking about the cloud provider secret :)

But since I injected that snippet about the SSH key, it gets confusing. I'll reword it.